internal/cmd/auth_listapitokens.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	apiTokensCmd.AddCommand(listApiTokensCmd)
+}
+
+var listApiTokensCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List api tokens.",
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.SilenceUsage = true
+
+		client, err := createTursoClient()
+		if err != nil {
+			return err
+		}
+
+		apiTokens, err := getApiTokens(client)
+		if err != nil {
+			return err
+		}
+
+		data := [][]string{}
+		for _, apiToken := range apiTokens {
+			data = append(data, []string{apiToken.Name, fmt.Sprint(apiToken.Owner), fmt.Sprint(apiToken.PubKey)})
+		}
+		printTable([]string{"Name", "Owner", "PubKey"}, data)
```

Comment:
@athoscouto I am not sure about the API tokens use cases, Which field do we need to show in the list command? 
And the PubKey is returning empty, not sure if this is the current behavior or if I did something wrong with the unmarshal.

Reply:
Got it.
I updated the PR with your points.

About the PubKey, I saw why it is returning empty, it is defined with `json:"-"` in the API.
Do you think we need it here? 
If so, I can open a PR changing it there.

Reply:
We can move forward without it. If somebody asks, we add it.